packages/integration-sdk-runtime/src/api/index.ts

```diff
@@ -52,7 +52,7 @@ export function createApiClient({
   return new Alpha({
     baseURL: apiBaseUrl,
     headers,
-    retry: retryOptions,
+    retry: retryOptions ?? {},
```

Comment:
Is there an applicable test that would've caught this and we should consider updating now?

Reply:
It would have been nice to catch this in a test. This is where `Alpha` sets defaults: [relevant code](https://github.com/lifeomic/alpha/blob/9adf37ed32a030990e564d41cd09607fc7e6c9aa/src/adapters/response-retry.js#L56). The client uses a response interceptor on the response and if there is an error it will retry. What actually is assigned to `retry` isn't accessible outside this scope. It might be indirectly testable by making a test where the client needs to retry?

Reply:
That makes sense. It's probably not worth the effort at this point is what I'm kinda coming to see now.